category/traversable/instances.lean

```diff
@@ -0,0 +1,249 @@
+/-
+Copyright (c) 2018 Simon Hudon. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Author: Simon Hudon
+
+Instances of `traversable` for types from the core library
+-/
+
+import category.traversable.basic
+import category.basic
+import category.functor
+import category.applicative
+
+universe variables u v
+
+open function
+
+section identity
+
+open function functor
+
+variables {f f' : Type u → Type u}
+variables [applicative f] [applicative f']
+
+def id.traverse {α β : Type u} (g : α → f β) : id α → f (id β) := g
```

Comment:
I don't think we need this function. You could just write `id` on the next line.